src/FsAutoComplete/FsAutoComplete.fsproj

```diff
@@ -68,17 +64,12 @@
     <TargetsForTfmSpecificBuildOutput>
       $(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
-
   <!-- workaround for not being able to have p2p dependencies in tool output dir
   https://github.com/nuget/home/issues/3891#issuecomment-377319939 -->
-  <Target
-      Name="CopyProjectReferencesToPackage"
-      DependsOnTargets="ResolveReferences">
+  <Target Name="CopyProjectReferencesToPackage" DependsOnTargets="ResolveReferences">
     <ItemGroup>
-      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths -> WithMetadataValue('ReferenceSourceTarget', 'ProjectReference'))" />
+      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths -&gt; WithMetadataValue('ReferenceSourceTarget', 'ProjectReference'))" />
```

Comment:
Hmm I'm unsure about this change. I think some tool made this change for me. @baronfel, should I revert?

Reply:
Yes, this should be reverted